### ISSUE TITLE
Broken torrent trackers: ThePirateBay and PrivateHD search providers

### ISSUE THREAD

**Issue by neo-neo1** [pymedusa/Medusa#9891]
These 2 trackers have been broken under Medusa for quite some time. "Error while trying to test provider for results" is the error given when I click the button "Test for results" under the "Provider Options & Feel" tab

**Comment by neo-neo1** [comment]
I edited my comment with how to fix Medusa's broken support for PrivateHD. Not a dev or I would. Please fix Medusa's broken support for this popular tracker. Thank you devs for maintaining this indefensible project. 

P.S. If someone could look into fixing ThePirrateBay it would be appreciated as well.

**Comment by 007revad** [comment]
There seems to be others trackers that give the "Error while trying to test provider for results" error when I click the button "Test for results" under the "Provider Options & Feel" tab. This error occurs for public trackers that were all working a short while ago.

ThePirateBay
Rarbg
LimeTorrents

Today I changed the primary DNS server setting on my NAS (where Medusa is installed) from 9.9.9.9 to 1.1.1.1 and now Rarbg is working again. But ThePirateBay and LimeTorrents still won't work.

BTW I get the same error for private trackers (that used to work) like TorrentDay (though I have TD working in Jackett).

TorrentBytes gives a different error: 
"Save ChangesTest for results TypeError: Cannot read properties of undefined (reading 'status')"

On the plus side, while testing all this I noticed that 3 of my private trackers that I disabled ages ago when they stopped working are now working again  :o)

**Comment by neo-neo1** [comment]
> There seems to be others trackers that give the "Error while trying to test provider for results" error when I click the button "Test for results" under the "Provider Options & Feel" tab. This error occurs for public trackers that were all working a short while ago.
> 
> ThePirateBay
> Rarbg
> LimeTorrents
> 
> Today I changed the primary DNS server setting on my NAS (where Medusa is installed) from 9.9.9.9 to 1.1.1.1 and now Rarbg is working again. But ThePirateBay and LimeTorrents still won't work.
> 
> BTW I get the same error for private trackers (that used to work) like TorrentDay (though I have TD working in Jackett).
> 
> TorrentBytes gives a different error:
> "Save ChangesTest for results TypeError: Cannot read properties of undefined (reading 'status')"
> 
> On the plus side, while testing all this I noticed that 3 of my private trackers that I disabled ages ago when they stopped working are now working again :o)

Nope, others work fine. Only ThePiratetBay and PrivateHD (as the title) don't work - and I already found the reason for PrivateHD, just need attention from a Medusa dev. Rarbg, LimeTorrents, and TorrentDay work fine with no issue. I'm a heavy daily user. You need to troubleshoot your setup as it is not Medusa related.
